roles/contiv/tasks/default_network.yml

```diff
@@ -1,16 +1,18 @@
+---
+
+# Is there a reason shell is used here instead of command?
 - name: Contiv | Wait for netmaster
-  action: shell netctl tenant ls
+  shell: 'netctl --netmaster "http://{{ inventory_hostname }}:{{ netmaster_port}}" tenant ls'
```

Comment:
Addint the explicit command line argument for the netmaster URL should not be necessary since the DNS entry for "netmaster" is already supposed to be populated in all the master nodes by a task that is currently in the contiv/tasks/netmaster.yml file (see "Netmaster | Build hosts file"). However that task seems erroneous at the moment so it probably doesn't hurt to use the explicit URL here ... hence will accept it.